website/docs/r/application.html.markdown

```diff
@@ -20,7 +20,7 @@ resource "azuread_application" "example" {
   name                       = "example"
   homepage                   = "https://homepage"
   identifier_uris            = ["https://uri"]
-  reply_urls                 = ["https://replyurl"]
+  reply_urreply_urls                 = ["https://replyurl"]ls                 = ["https://replyurl"]
```

Comment:
Is this right?

Reply:
no lol none of this is - i need to fix this pr for sure as changes snuk in. thanks